lib/dialects/abstract/query-generator.js

```diff
@@ -407,20 +407,21 @@ class QueryGenerator {
   /**
    * Returns an update query using arithmetic operator
    *
-   * @param {string} operator      String with the arithmetic operator (e.g. '+' or '-')
-   * @param {string} tableName     Name of the table
-   * @param {Object} attrValueHash A hash with attribute-value-pairs
-   * @param {Object} where         A hash with conditions (e.g. {name: 'foo'}) OR an ID as integer
+   * @param {string} operator                    String with the arithmetic operator (e.g. '+' or '-')
+   * @param {string} tableName                   Name of the table
+   * @param {Object} where                       A plain-object with conditions (e.g. {name: 'foo'}) OR an ID as integer
+   * @param {Object} incrementAmountsByField     A plain-object with attribute-value-pairs
+   * @param {Object} extraAttributesToBeUpdated  A plain-object with attribute-value-pairs
    * @param {Object} options
-   * @param {Object} attributes
+   * 
+   * @private
```

Comment:
Note: here I am simply making explicit the fact that this method is private (since the whole namespace is already `@private`)